hack/generator/pkg/jsonast/schema_abstraction_openapi.go

```diff
@@ -7,12 +7,12 @@ package jsonast
 
 import (
 	"fmt"
-	"io/ioutil"
-	"net/url"
-
 	"github.com/go-openapi/jsonpointer"
 	"github.com/go-openapi/spec"
 	"github.com/pkg/errors"
+	"io/ioutil"
```

Comment:
These imports should be at the top - the general pattern is one block for standard lib packages, one for external packages, one for packages in the current project. Might be an IDE setup issue?

Reply:
I didn't change them manually. I'll put them back. GoLand uses `go fmt` so I assume the changes were done by that.

Reply:
my formatter always puts stdlib packages at the top. is there some kind of config option?

Reply:
looks like the VSCode formatter is `goimports` not just `go fmt`

Reply:
GoLand configuration here: https://www.jetbrains.com/help/go/integration-with-go-tools.html#goimports

Reply:
and automatic run setting: https://stackoverflow.com/a/45590498/10311